app/utils/CheckYourAnswersHelper.scala

```diff
@@ -22,12 +22,19 @@ import pages._
 import pages.engineering._
 import pages.healthcare._
 import pages.transport._
-import viewmodels.AnswerRow
+import viewmodels.{AnswerRow, RepeaterAnswerRow, RepeaterAnswerSection}
```

Comment:
why new imports when there not used?

Reply:
Not 100% sure think this came from another merge have removed